package.json

```diff
@@ -13,7 +13,7 @@
     "lerna:deploy": "node lernaDeploy.js"
   },
   "devDependencies": {
-    "@commitlint/cli": "^8.3.5",
+    "@commitlint/cli": "^11.0.0",
```

Comment:
although a breaking change, this is a dev dependency...

Reply:
we should be good